app/controllers/admin/overview_controller.rb

```diff
@@ -2,8 +2,10 @@ module Admin
   class OverviewController < Admin::ApplicationController
     layout "admin"
     def index
-      @open_abuse_reports_count =
```

Comment:
Was this dead code?

Reply:
I believe it was moved to `app/services/admin/data_counts.rb`.